docs/workloads/azure/backend/netcore/architecture/architecture_overview_netcore.md

```diff
@@ -18,101 +18,52 @@ keywords:
   - stacks
 ---
 
+### Domain-Driven Design (DDD) Overview
 
-## DDD Architecture Overview
+For a comprehensive understanding of Domain-Driven Design, refer to Martin Fowler's notes on [Domain-Driven Design](https://martinfowler.com/bliki/DomainDrivenDesign.html).
 
-![Solution Diagram](/img/cqrs_diagram.png)
```

Comment:
Was the diagram not adding anything?

Reply:
I believe it is better suited in a section about the solution structure and not a general information section on DDD